src/prog.js

```diff
@@ -24,7 +24,8 @@ export default handler => {
 		.option('--external', `Specify external dependencies, or 'none'`)
 		.option('--globals', `Specify globals dependencies, or 'none'`)
 		.example('microbundle --globals react=React,jquery=$')
-		.option('--compress', 'Compress output using Terser', null)
+		.option('--compress', 'Compress output using Terser (default)', null)
+	        .option('--no-compress', 'Disable output compressing', null)
```

Comment:
sade automatically adds that option, but yes it's confusion

Reply:
and ci is complaining about the indent here 🙃

Reply:
yeah, took me diving in the source to figure how to disable compress, as `--compress=false` and similar doesn't work!

Reply:
Ok, attempted to fix, hope it worked out (creating this PR from the GitHub interface)

Reply:
Still want to make it the default, maybe soon